tools/packaging/common/envoy_bootstrap.json

```diff
@@ -483,7 +483,7 @@
                     ]
                   },
                   "http_filters": [{
-                    "name": "envoy.router",
+                    "name": "envoy.filters.http.router",
```

Comment:
it will be good that the same name should be updated everywhere to avoid confusion.
https://github.com/istio/istio/blob/master/pkg/test/env/envoy_conf.go#L81